selectrum-prescient.el

```diff
@@ -117,6 +117,73 @@ For use on `selectrum-candidate-selected-hook'."
     (remove-hook 'selectrum-candidate-inserted-hook
                  #'selectrum-prescient--remember)))
 
+;;;; Commands
+(defvar selectrum-prescient-filter-toggle-map (make-sparse-keymap)
+  "A keymap of commands for toggling Prescient filters in Selectrum.
+The toggling of commands is temporary and does not affect the
+default filtering settings determined by `prescient-filter-method'.")
+;; Create a binding similar to the Isearch toggles.
+(defvar selectrum-minibuffer-map)
+(define-key selectrum-minibuffer-map
+  "\M-s" selectrum-prescient-filter-toggle-map)
```

Comment:
```suggestion
  (kbd "M-s") selectrum-prescient-filter-toggle-map)
```

Reply:
Is it possible to make it so that this binding is only activated when `selectrum-prescient-mode` is enabled? While it's easy to do it this way, it's generally considered bad practice for simply loading a package to cause behavior changes.

I'd be happy to spend some time thinking about a good way of making this happen, if you're not sure.